tools/pycbc_test_suite.sh

```diff
@@ -69,6 +70,24 @@ else
 fi
 popd
 
+## Run inference on GW150914 data
```

Comment:
Wouldn't it be clearer to have a dedicated script instead of running sed? This seems like it might be fragile to me.

Reply:
@ahnitz I had a script to run this unit test on in the first commit of this PR. But @cdcapano wanted to edit and use the scripts in the examples directory so that those are tested as well when this runs. @cdcapano can you comment? 

If a dedicated script is desired let me know the changes you want in what I had initially.